src/main/java/mate/academy/spring/mapper/impl/response/ShoppingCartResponseMapper.java

```diff
@@ -0,0 +1,23 @@
+package mate.academy.spring.mapper.impl.response;
+
+import java.util.stream.Collectors;
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.ShoppingCart;
+import mate.academy.spring.model.Ticket;
+import mate.academy.spring.model.dto.response.ShoppingCartResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ShoppingCartResponseMapper
+        implements DtoResponseMapper<ShoppingCartResponseDto, ShoppingCart> {
+    @Override
+    public ShoppingCartResponseDto toDto(ShoppingCart object) {
```

Comment:
similar problem with the variable name, in my opinion it is better to give more accurate names